query-connector/src/app/query/components/customizeQuery/CustomizeQueryNav.tsx

```diff
@@ -1,14 +1,14 @@
 import { DibbsConceptType } from "@/app/constants";
 import styles from "./customizeQuery.module.scss";
 import CustomizeQueryBulkSelect from "./CustomizeQueryBulkSelect";
-import { ConceptTypeToVsNameToVsGroupingMap } from "@/app/utils/valueSetTranslation";
+import { temp__ConceptTypeToVsNameToVsGroupingMap } from "@/app/utils/valueSetTranslation";
```

Comment:
_note:_ I added this prefix where necessary to avoid breaking Query Building, since I didn't touch that side of things